templates/mobile/activation/activation.1.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "activation",
+  "properties": {
+    "identifier": {
+      "type": "string",
+      "description": "hashed version of Google ID"
+    },
+    "client_id": {
+      "type": "string",
+      @COMMON_PATTERN_UUID_1_JSON@,
+      "description": "telemetry client_id, included only if identifer is empty"
+    },
+    "manufacturer": {
+      "type": "string",
+      "description": "Build.MANUFACTURER"
+    },
+    "model": {
+      "type": "string",
+      "description": "Build.MODEL"
+    },
+    "device": {
+      "type": "string"
+    },
+    "distribution_id": {
+      "type": "string"
+    },
+    "locale": {
+      "type": "string",
+      "description": "application locale, such as 'en-US'"
+    },
+    "os": {
+      "type": "string"
+    },
+    "osversion": {
+      "type": "string"
+    },
+    "created": {
+      "type": "string"
+    },
+    "profile_date": {
+      "type": "integer"
+    },
+    "tz": {
+      "type": "string"
+    },
+    "app_name": {
+      "enum": [
+        "Fennec"
+      ]
+    },
```

Comment:
The "enum" construction here means any value other than "Fennec" will cause the ping to fail validation and be rejected.

We could drop this `app_name` field entirely if we never expect this ping to be used by any other application.

Reply:
Changed this back to just a plain string. Seems safer that way.